app/views/users/accounts/role_not_found.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= h1 "You do not have permission to use this service" %>
+
+<% if @cis2_info[:has_other_roles] %>
+  <%= govuk_button_to "Change role", user_cis2_omniauth_authorize_path, params: { change_role: true } %>
+<% end %>
+
+<h2 class="nhsuk-heading-m">Contact your registration authority</h2>
+
+<p class="nhsuk-body">
```

Comment:
```suggestion
<p>
```

Reply:
Ya, you commented this on the last PR, but this is copied from the design. Is it extraneous there too?

Reply:
Yup, it's a CSS class that isn't defined and has no rules. cc @paulrobertlloyd